eng/Versions.props

```diff
@@ -14,7 +14,7 @@
     <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
     <!-- F# Version components -->
     <FSMajorVersion>7</FSMajorVersion>
-    <FSMinorVersion>0</FSMinorVersion>
+    <FSMinorVersion>1</FSMinorVersion>
```

Comment:
@vzarytovskii , @KevinRansom  : Like this?
Or did I oversee some other place(s), where I should make a change as well?

Reply:
@KevinRansom told me that in the case of waiting for next preview version, we should leave the versioning scheme as is, it will cover for this.